lib/auth/auth.go

```diff
@@ -33,3 +33,10 @@ func NewToken(authToken string) (*Token, error) {
 	}
 	return &at, nil
 }
+
+func (t *Token) String() string {
+	if t == nil {
```

Comment:
This check make no sense - you can't call a member function for nil object. In other words, if t==nil - you'll get panic on function call before even reaching this check.